unpack_action.go

```diff
@@ -51,7 +51,12 @@ func (pf *UnpackAction) Verify(context *DebosContext) error {
 
 func (pf *UnpackAction) Run(context *DebosContext) error {
 	pf.LogStart()
-	infile := path.Join(context.artifactdir, pf.File)
+
+	//Trying to get a filename from exports first
+	infile, found := context.origins[pf.File]
```

Comment:
We shouldn't overload File to mean origin. Also the file to unpack could be in a downloaded tarball, so you stil lwant Origin

So this chunk should instead add an Origin key - File is optional when the Origin is a named action which produces a file (e.g. download without unpack set) or when given it's the path of the file to unpack within the origin (by default the artifact dir)

Reply:
done